### PR TITLE
Reduce Coupling from Unify and Bootstrap

### PR DESCRIPTION
**Pull request by Dan-Bird**
This PR begins to uncouple the website from using the Unify and Bootstrap libraries. 

Some dead code has been tidied up. A section of the Homepage has been cleaned to follow the BEM naming convention and reduce coupling from helper classes from the above mentioned libraries.

### CODE REVIEW

**Comment by chriseyre2000** [comment]
❤️ ❤️ ❤️ ❤️ ❤️ 

Killing Unify!

File: src/index.html [hunk 1]
```diff
@@ -84,7 +84,7 @@ <h2 class="h3">{% t index-activity.title %}</h2>
 
 
 <!-- Partner panel -->
-<div class="section gray-section">
+<section class="section gray-section">
```
Comment (mattgraygithub) [inline src/index.html:87]: Guessing we're going to do more with these classes to make them more descriptive, and then remove the comments? Looks great so far though!

**Review by mattgraygithub — APPROVED** [review]
Looking good to me 👍

File: src/index.html (hunk 1 above)
Comment (Dan-Bird) [reply src/index.html:87]: Yeah exactly. Once they follow the BEM convention they'll already be descriptive and the comments would then be redundant

**Pull request merged**